sql/report_queries/fund/fund_expenditures_by_po_line/fund_expenditures_by_po_line

```diff
@@ -0,0 +1,156 @@
+/* fund_expenditures_by_po_line
+ 
+PURPOSE
+This report shows fund expenditures by purchase order line. 
+For each purchase order line number, the fund data attributes are  
+shown alongside the purchase order line transaction amount. Fund expenditures 
+can be viewed by fund group, fund name, fund type, fund code, 
+purchase order line format, transaction type, expense class, and fiscal year.
+There can be more than one fund distribution for each purchase order line.
+
+The transaction_fund_code is the fund code associated with each 
+purchase order line transaction. This column captures "Payment" and
```

Comment:
Shouldn't this read invoice line transaction instead of purchase order line transaction